draft-ietf-quic-http.md

```diff
@@ -860,20 +860,26 @@ details.
 The "chunked" transfer encoding defined in Section 4.1 of {{!RFC7230}} MUST NOT
 be used.
 
-Trailing header fields are carried in an additional header block following the
-body. Senders MUST send only one header block in the trailers section;
-receivers MUST discard any subsequent header blocks.
+Trailing header fields are carried in an additional HEADERS frame following the
+body. Senders MUST send only one HEADERS frame in the trailers section;
+receivers MUST discard any subsequent HEADERS frames.
 
 A response MAY consist of multiple messages when and only when one or more
 informational responses (1xx, see {{!RFC7231}}, Section 6.2) precede a final
 response to the same request.  Non-final responses do not contain a payload body
 or trailers.
 
 An HTTP request/response exchange fully consumes a bidirectional QUIC stream.
-After sending a request, a client closes the stream for sending; after sending a
-final response, the server closes the stream for sending and the QUIC stream is
-fully closed.  Requests and responses are considered complete when the
-corresponding QUIC stream is closed in the appropriate direction.
+After sending a request, a client MUST close the stream for sending; after
+sending a final response, the server MUST close the stream for sending. At
+this point, the QUIC stream is fully closed.
```

Comment:
To @martinduke's comment, maybe it is enough to say "A message is terminated when the stream is closed."  Instead of all these words.

Reply:
Oh, and now I realize that this is what it said before...  Not sure whether this is an improvement then.

Reply:
Yeah, the original provoking issue was that he didn't FIN, the server didn't respond, and both are doing totally spec-compliant things.  With this language, it's at least clarified that while the server SHOULD process that request on receipt, it isn't required to until the stream closes, so the client needs to close promptly.